examples/complex/test/AppManager.test.js

```diff
@@ -0,0 +1,105 @@
+const OwnedUpgradeabilityProxy = artifacts.require('zos-lib/contracts/upgradeability/OwnedUpgradeabilityProxy.sol');
+const UpgradeabilityProxyFactory = artifacts.require('UpgradeabilityProxyFactory');
+const Package = artifacts.require('Package');
+const AppDirectory = artifacts.require('AppDirectory');
+const ContractDirectory = artifacts.require('ContractDirectory');
+const AppManager = artifacts.require('PackagedAppManager');
+const MintableERC721Token = artifacts.require('MintableERC721Token');
+
+const DonationsV1 = artifacts.require('DonationsV1');
+const DonationsV2 = artifacts.require('DonationsV2');
+
+const decodeLogs = require('zos-lib').decodeLogs;
+const encodeCall = require('zos-lib').encodeCall;
+const validateAddress = require('./helpers/validateAddress.js');
+const shouldBehaveLikeDonations = require('./Donations.behavior.js');
+const shouldBehaveLikeDonationsWithTokens = require('./DonationsWithTokens.behavior.js');
+const should = require('chai').should();
+const deploy = require('../index.js');
+
+contract('AppManager', (accounts) => {
+
+  before(function() {
+    this.owner = accounts[1];
+    this.donor1 = accounts[2];
+    this.wallet = accounts[4];
```

Comment:
1. I am aware of that trick, and I intentionally try to avoid it. It's a matter of personal preference. Let me know if we want to precede consistency with opinions and I'll change it.
2. Totally. Changed.